src/transformers/models/bridgetower/modeling_bridgetower.py

```diff
@@ -0,0 +1,1208 @@
+            Tuple of `torch.FloatTensor` (one for the output of the embeddings, if the model has an embedding layer, +
+            one for the output of each layer) of shape `(batch_size, sequence_length, hidden_size)`.
+
+            Hidden-states of the model at the output of each layer plus the optional initial embedding outputs.
+        attentions (`tuple(torch.FloatTensor)`, *optional*, returned when `output_attentions=True` is passed or when `config.output_attentions=True`):
+            Tuple of `torch.FloatTensor` (one for each layer) of shape `(batch_size, num_heads, sequence_length,
+            sequence_length)`.
+
+            Attentions weights after the attention softmax, used to compute the weighted average in the self-attention
+            heads.
+    """
+
+    text_feats: torch.FloatTensor = None
+    image_feats: torch.FloatTensor = None
+    pooler_output: torch.FloatTensor = None
+    hidden_states: Optional[Tuple[torch.FloatTensor]] = None
+    attentions: Optional[Tuple[torch.FloatTensor]] = None
+
+
+@add_start_docstrings(
+    "The bare BridgeTower Model transformer outputting BridgeTowerModelOutput object without any specific head on"
+    " top.",
+    BRIDGETOWER_START_DOCSTRING,
+)
+class BridgeTowerModel(BridgeTowerPreTrainedModel):
+    def __init__(self, config):
+        super().__init__(config)
+        self.is_clip = True if config.vit_layers > 0 else False
+
+        if "roberta" in config.tokenizer:
+            self.tokenizer_config = RobertaConfig(
+                vocab_size=config.vocab_size,
+                hidden_size=config.hidden_size,
+                num_hidden_layers=config.num_hidden_layers,
+                num_attention_heads=config.num_attention_heads,
+                intermediate_size=config.hidden_size * config.mlp_ratio,
+                max_position_embeddings=config.max_text_len,
+                hidden_dropout_prob=config.drop_rate,
+                attention_probs_dropout_prob=config.drop_rate,
+            )
+        else:
+            raise ValueError("Incorrect value of tokenizer. Currently roberta tokenizer is supported")
+
+        if config.cross_modal_transform_shared:
+            self.cross_modal_text_transform = nn.Linear(config.input_text_embed_size, config.hidden_size)
+            self.cross_modal_image_transform = nn.Linear(config.input_image_embed_size, config.hidden_size)
+        else:
+            self.cross_modal_text_transform = nn.ModuleList(
+                [nn.Linear(config.input_text_embed_size, config.hidden_size) for _ in range(config.num_hidden_layers)]
+            )
+            self.cross_modal_image_transform = nn.ModuleList(
+                [nn.Linear(config.input_image_embed_size, config.hidden_size) for _ in range(config.num_hidden_layers)]
+            )
+        self.cross_modal_text_transform.apply(self._init_weights)
+        self.cross_modal_image_transform.apply(self._init_weights)
```

Comment:
I don't think this is needed, you'll probably just need to call `self.post_init` at the end of the init function and should be fine

Reply:
Handled! Thanks for the feedback.